cmake/projects/protobuf/hunter.cmake

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2014-2015, David Linten
+# All rights reserved.
+
+if(DEFINED HUNTER_CMAKE_PROJECTS_PROTOBUF_HUNTER_CMAKE_)
+  return()
+else()
+  set(HUNTER_CMAKE_PROJECTS_PROTOBUF_HUNTER_CMAKE_ 1)
+endif()
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    protobuf
+    VERSION
+    "v3.0.0-hunter-p1"
+    URL
+    "https://github.com/dlinten/protobuf/archive/v3.0.0-hunter-p1.tar.gz"
```

Comment:
> please send your local patches there to branch hunter

Even if you have permissions to pull to branch directly it's better to send pull request for review reasons.